vunit/vhdl_parser.py

```diff
@@ -43,6 +43,7 @@ class VHDLDesignFile(object):  # pylint: disable=too-many-instance-attributes
     """
     Contains VHDL objects found within a file
     """
+
```

Comment:
Are these spaces additions required? If so, it might be worth making a commit with these only.

Reply:
This is the result of some code formatting settings I have. While we don't have any such rules I think both autopep8 and pydev's formatter will add this line so I guess we can call this good practice. I think we should consider a standard formatter. Meanwhile we could apply the boy scout rule

Reply:
Your are correct. I executed `autopep8 vunit/vhdl_parser.py` and it introduced these spaces, among other changes.

I think it'd be good to have any standard formatter. Being able to execute it either in a container or as a git hook would be really helpful.